scss/_global.scss

```diff
@@ -20,35 +20,50 @@ $global-width: rem-calc(1200) !default;
 /// @type Number
 $global-lineheight: 1.5 !default;
 
-/// Colors used for buttons, callouts, links, etc. There must always be a color called `primary`.
+/// Colors available by using `get-color()`.
 /// @type Map
-$foundation-palette: (
-  primary: #1779ba,
-  secondary: #767676,
-  success: #3adb76,
-  warning: #ffae00,
-  alert: #cc4b37,
+$foundation-colors: (
```

Comment:
I don't know if `_global.scss` is the best place for this. We should rather put it in `util/_color.scss`.